src/pages/Explorer/Category/ExplorerCategory.svelte

```diff
@@ -1,98 +1,60 @@
+  import { onMount } from 'svelte'
   import Range from 'webkit/ui/Range.svelte'
   import Category from './Category.svelte'
-  import ChartLayoutItem from '../Layouts/ChartLayoutItem.svelte'
+  import LayoutItem from '../Layouts/LayoutItem.svelte'
   import AssetSelector from '../Components/AssetSelector.svelte'
-  import { EntityType } from '../const'
-
-  export let items = [
-    {
-      id: 0,
-      type: 'chart',
-      title: 'Protocol - Fundamental comparison (TVL, User, Develel)',
-      comments: 0,
-      votes: 0,
-      user: { username: 'test' },
-      assets: [
-        { slug: 'bitcoin' },
-        { slug: 'ethereum' },
-        { slug: 'bitcoin-cash' },
-        { slug: 'bitcoin' },
-      ],
-    },
-  ]
-
-  const RANGES = {
-    '24h': '1d',
-    '7d': '7d',
-    '1m': '30d',
-    '6m': '183d',
-    '1y': '1y',
-    'All time': 'all',
-  }
-
+  import EmptyState from '../Components/EmptyState.svelte'
+  import TypeSelector from '../Components/TypeSelector.svelte'
+  import { getItems, EntityType, RANGES, MenuItem } from '../const'
+
+  export let activeMenu
+  let items = []
+  let range = ''
+  let assets = []
   let types = new Set(Object.values(EntityType))
+  let page = 1
 
-  const toggleType = (type) => {
-    if (types.has(type)) {
-      types.delete(type)
-    } else {
-      types.add(type)
-    }
-    types = types
+  function fetch() {
+    const voted = activeMenu === MenuItem.LIKES
+    const currentUserDataOnly = activeMenu === MenuItem.MY_CREATIONS
+    const args = { voted, range, page, currentUserDataOnly }
+    getItems(args).then((res) => (items = res))
   }
-</script>
-
-<Category title="Explorer" {items}>
-  <div slot="header" class="controls row mrg-a mrg--l">
-    <!-- TODDO add `onChange` prop -->
-    <Range items={Object.keys(RANGES)} selectedIndex="5" />
-
-    <AssetSelector />
-
-    <!-- TODO: move to Types.svelte -->
-    <Tootlip on="click" activeClass="$style.active" align="end">
-      <div slot="trigger" class="btn-2 btn--s">Types: {types.size}</div>
 
-      <div slot="tooltip" class="tooltip">
-        <div class="caption txt-m mrg-s mrg--l mrg--b c-waterloo">Types</div>
-        {#each Object.values(EntityType) as type, index (index)}
-          <div
-            class="btn-ghost row v-center"
-            on:click={() => toggleType(type)}
-            style="fill: {type.color}"
-          >
-            <Svg id={type.icon} w="16" class="mrg-s mrg--r" />
-            {type.label}
-            <Checkbox isActive={types.has(type)} class="mrg-a mrg--l" />
-          </div>
-        {/each}
-      </div>
-    </Tootlip>
-  </div>
-
-  <svelte:fragment let:item>
-    {#if item.type === 'chart'}
-      <ChartLayoutItem {item} showActions={true} />
-    {/if}
-  </svelte:fragment>
-</Category>
-
-<style>
-  .active {
-    --border: var(--green);
-  }
-
-  .tooltip {
-    width: 176px;
-    padding: 12px 12px 16px;
-  }
+  onMount(fetch)
+  $: activeMenu, range, assets, types, page, fetch()
```

Comment:
`onMount` is not needed, because of the reactive statement below
```suggestion
  $: activeMenu, range, assets, types, page, fetch()
```